src/main/java/org/terasology/launcher/updater/LauncherUpdaterFactory.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2016 MovingBlocks
```

Comment:
Ah, right :)

Should I update it on all the files I worked on, or only on the new ones created on this branch?